src/components/testCaseLanding/qdm/TestCaseCoverage/CoverageTabs/CoverageTab.tsx

```diff
@@ -1,42 +1,63 @@
 import React from "react";
 interface Props {
-  population: string;
-  definitionText: any;
+  definition: string;
+  definitionResults: any;
+  cqlDefinitionCallstack: any;
+  groupCoverageResult: any;
 }
 import { Accordion } from "@madie/madie-design-system/dist/react";
+import parse from "html-react-parser";
+import { StatementCoverageResult } from "../../../../../util/cqlCoverageBuilder/CqlCoverageBuilder";
+import { isPopulation } from "../../../../../util/GroupCoverageHelpers";
+import DefinitionsUsedSection from "../../../../editTestCase/groupCoverage/DefinitionsUsedSection";
+import { isNil } from "lodash";
 import "twin.macro";
 import "styled-components/macro";
 
-const CoverageTab = ({ population, definitionText }: Props) => {
-  return population !== "Functions" &&
-    population !== "Used" &&
-    population !== "Unused" ? (
+const CoverageTab = ({
+  definition,
+  definitionResults,
+  cqlDefinitionCallstack,
+  groupCoverageResult,
+}: Props) => {
+  const getCoverageResult = (coverageResult: StatementCoverageResult) => {
+    if (isNil(coverageResult)) {
+      return "No results available";
+    }
+
+    return [
+      parse(`<pre><code>${coverageResult.html}</code></pre>`),
+      isPopulation(definition) && cqlDefinitionCallstack && (
+        <DefinitionsUsedSection
+          result={definitionResults[0]}
+          cqlDefinitionCallstack={cqlDefinitionCallstack}
+          groupCoverageResult={groupCoverageResult}
+        />
+      ),
+    ];
+  };
+  return definition !== "Functions" &&
+    definition !== "Used" &&
+    definition !== "Unused" ? (
     <div
       style={{ maxWidth: "1300px" }}
-      data-testid={`${population}-population`}
+      data-testid={`${definition}-population`}
     >
-      <Accordion title={population} isOpen={false}>
-        <pre data-testId={`${population}-population-text`}>
-          {definitionText.text}
+      <Accordion title={definition} isOpen={false}>
+        <pre data-testId={`${definition}-population-text`}>
+          {definitionResults.map((results) => getCoverageResult(results))}
```

Comment:
this will give you html for population as well as its used definitions.

e.g if my Initial population is:
```
define "Initial Population":
  "Qualifying Encounters"
```  

then it will result into initial population plus it's used definition something like this:
```
define "Initial Population":
  "Qualifying Encounters"

Definition(s) Used


define "Qualifying Encounters":
  ["Encounter, Performed": "Encounter Inpatient"] Encounter
    where Encounter.relevantPeriod ends during "Measurement Period"
```
we don't want to show used definitions for populations here. should display only population definition. `getCoverageResult` should be updated to not return used defines for population